src/Dropdown/index.jsx

```diff
@@ -95,10 +115,22 @@ DropdownToggle.defaultProps = {
   bsPrefix: 'dropdown-toggle',
 };
 
+Dropdown.Item = ({ className, ...otherProps }) => (
```

Comment:
Let's create this function with a `forwardRef` in case the consumer needs to pass a ref to this component, similar to how the original `Dropdown.Item` is [implemented](https://github.com/react-bootstrap/react-bootstrap/blob/v1.6.6/src/DropdownItem.tsx#L77) by react-bootstrap as well as other components in Paragon.

Reply:
+1